specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5535,7 +5535,7 @@
             "url": "https://docs.microsoft.com/rest/api/searchservice/Create-Data-Source"
           },
           "type": "string",
-          "description": "The connection string for the datasource."
+          "description": "The connection string for the datasource. Set to '<unchanged>' if you do not want the connection string updated."
```

Comment:
We don't need to maintain old Swagger specs. Modifying 2020-06-30 and 2020-06-30-preview would suffice.

Reply:
reverted